dubbo-spring-boot/dubbo-spring-boot-actuator/src/main/resources/META-INF/dubbo-endpoints-default.properties

```diff
@@ -9,6 +9,43 @@ management.endpoint.dubboconfigs.enabled = true
 management.endpoint.dubboservices.enabled = false
 management.endpoint.dubboreferences.enabled = false
 management.endpoint.dubboproperties.enabled = true
+management.endpoint.dubbohelp.enabled = true
+management.endpoint.dubbocd.enabled = false
+management.endpoint.dubbocount.enabled = false
+management.endpoint.dubbopwd.enabled = false
+management.endpoint.dubboinvoke.enabled = false
+management.endpoint.dubboselect.enabled = false
+management.endpoint.dubboready.enabled = true
+management.endpoint.dubbols.enabled = true
+management.endpoint.dubbostartup.enabled = true
+management.endpoint.dubbops.enabled = true
+management.endpoint.dubboversion.enabled = true
+management.endpoint.dubbogetaddress.enabled = true
+management.endpoint.dubbogetconfig.enabled = true
+management.endpoint.dubbometrics.enabled = true
+management.endpoint.dubbometrics_default.enabled = true
+management.endpoint.dubbopublishMetadata.enabled = true
+management.endpoint.dubboonline.enabled = true
+management.endpoint.dubboonlineapp.enabled = true
+management.endpoint.dubboonlineinterface.enabled = true
+management.endpoint.dubbooffline.enabled = true
+management.endpoint.dubboofflineapp.enabled = true
+management.endpoint.dubboofflineinterface.enabled = true
+management.endpoint.dubbologgerinfo.enabled = true
+management.endpoint.dubboswitchlogger.enabled = true
+management.endpoint.dubboswitchloglevel.enabled = true
+management.endpoint.dubbodisabledetailprofiler.enabled = true
+management.endpoint.dubbodisablesimpleprofiler.enabled = false
+management.endpoint.dubboenabledetailprofiler.enabled = true
```

Comment:
Can we generate this automatically

Reply:
Perhaps I can automatically generate using the information in this file, allowing configurations to be automatically added when new QoS features are introduced. These default configuration settings can also be left empty, and if they are empty, they will be enabled by default.
![image](https://github.com/user-attachments/assets/ad3fec5d-d006-4f31-a898-6cbb840a0a45)

Reply:
Yes, from this